src/app/app.component.ts

```diff
@@ -1,10 +1,31 @@
 import { Component } from '@angular/core';
+import { Router } from '@angular/router';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  styleUrls: ['./app.component.scss'],
 })
 export class AppComponent {
   title = 'boilerplate-frontend-orion-bootcamp';
+
+  constructor(private router: Router) {}
+
+  /**
+   * Check if the current route is different of '/' and '/recovery'
+   * @returns 'true' if it is different a route, 'false' otherwise
+   */
+  isAuthPage(): boolean {
```

Comment:
O nome da sua função não condiz com o que ela faz.

Pelo nome, está verificando se a página em questão, é a de login, mas nas condições, verifica se é diferente das rotas de login.

Se a ideia for verificar se são páginas que necessitam estar autenticadas, eu mudaria o nome da função pra alguma coisa do tipo `isPrivatePage`.

Lembrando ainda sobre o que foi mencionado no PR#9, as rotas vão sofrer uma alteração, então você pode apenas verificar se as rotas contém o caminho que deverão conter as rotas privadas.